### PR TITLE
Include `<cstdint>` in headers that use fixed width integer types.

### PR DESCRIPTION
**Pull request by mmuetzel**
The STL implementation from newer versions of GCC implicitly include `<cstdint>` in less headers.
Include that header explicitly in files that use fixed width integer types (like `int64_t`) to fix a compilation error with these versions of GCC.

### CODE REVIEW

**Review by jsandham — APPROVED** [review]

**Comment by mmuetzel** [comment]
Sorry if this is stupid question: What are the necessary steps after a PR gets approved here? Is someone else reviewing and merging the PR? (I'm not an authorized user in this repository.)

**Comment by jsandham** [comment]
> Sorry if this is stupid question: What are the necessary steps after a PR gets approved here? Is someone else reviewing and merging the PR? (I'm not an authorized user in this repository.)

Thanks for the contribution and there are no stupid questions =). Ill merge once I meet with the team today.

**Review by YvanMokwinski — APPROVED** [review]
LGTM

**Pull request merged**
